### ISSUE TITLE
Pretrained models COCO

### ISSUE THREAD

**Issue by FacuRoffet99** [microsoft/Swin-Transformer#255]
Hi, where can I find the weights for the models pretrained on COCO?
Thank you!

**Comment by impiga** [comment]
Hi, please check https://github.com/SwinTransformer/Swin-Transformer-Object-Detection .

**Comment by FacuRoffet99** [comment]
Thank you very much for your answer, but I have already checked that link before.
In that repository the pre-trained models of Mask R-CNN, Cascade Mask and RepPoints are available, but not those of HTC++. Do you know where I can find those?
